Makefile

```
@@ -0,0 +1,207 @@
+start:
+	@$(PNPM) start
+
+# Runs all tests for all applications.
+test-all:
+	@$(PNPM) test:all
+
+# Runs the pre-commit checks.
+precommit:
+	@$(PNPM) precommit
+
+.PHONY: help install update-deps prepare ruler-check ruler-apply dev dev-landing dev-web dev-docs build build-landing preview-landing build-web build-docs test test-ui test-coverage lint lint-strict check clean backend-build backend-run backend-test backend-clean start test-all precommit
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)